builtin/commit-graph.c

```diff
@@ -176,13 +177,15 @@ static int graph_write(int argc, const char **argv)
 			N_("maximum ratio between two levels of a split commit-graph")),
 		OPT_EXPIRY_DATE(0, "expire-time", &split_opts.expire_time,
 			N_("maximum number of commits in a non-base split commit-graph")),
+		OPT_BOOL(0, "progress", &opts.progress, N_("force progress reporting")),
```

Comment:
super-nit:  is there any interest in grouping this new option with the other `opts` rows above, rather than after the `split_opts` rows?